sunpy/net/vso/attrs.py

```diff
@@ -400,7 +400,12 @@ class Quicklook(_VSOSimpleAttr):
     Quicklook products are *not* searched by default.   Reference:
     documentation in SSWIDL routine vso_search.pro.
     """
-    pass
+    def __init__(self,value):
+        super(Quicklook,self).__init__(value)
+        if(self.value == True):
```

Comment:
I was more meaning why have the `== True` part.

Also `self.value` is the best option as you don't know what the super init is actually doing (in theory).